src/config-schema.ts

```diff
@@ -7,14 +7,24 @@ export const configSchema = {
     _description: "Uuid for orderType",
   },
   targetPatientDashboard: {
-    _type: Type.String,
-    _default: "Results Viewer",
+    redirectToResultsViewer: {
+      _type: Type.String,
+      _default: "Results Viewer",
+      _description:
+        "Redirects to Results Viewer in patient chart dashboard from lab app.",
+    },
+    redirectToOrders: {
+      _type: Type.String,
+      _default: "Orders",
+      _description:
+        "Redirects to Orders in patient chart dashboard from lab app.",
+    },
     _description:
       "The patient chart dashboard to navigate to from the lab app.",
   },
 };
 
 export type Config = {
   laboratoryOrderTypeUuid: string;
-  targetPatientDashboard: string;
+  targetPatientDashboard: ConfigSchema;
```

Comment:
I think making this an Object is more correct 
```suggestion
  targetPatientDashboard: Object;
```

Reply:
Hi @pirupius , I have updated this Change . Thank you.